packages/web/components/templates/homeFeed/HomeFeedContainer.tsx

```diff
@@ -259,11 +258,7 @@ export function HomeFeedContainer(props: HomeFeedContainerProps): JSX.Element {
         const username = viewerData?.me?.profile.username
         if (username) {
           setActiveCardId(item.node.id)
-          if (item.node.state === State.PROCESSING) {
-            router.push(`/${username}/links/${item.node.id}`)
-          } else {
-            router.push(`/${username}/${item.node.slug}`)
-          }
+          router.push(`/${username}/${item.node.slug}`)
```

Comment:
I am not sure if this works because slug can change after the item is saved.

Reply:
Changed to `id`. I will also add a refresh interval in slug page